packages/button/src/button-spinner.tsx

```diff
@@ -21,7 +21,7 @@ export const ButtonSpinner: React.FC<ButtonSpinnerProps> = (props) => {
   const {
     label,
     placement,
-    spacing,
+    spacing = "0.5",
```

Comment:
`0.5` refers to a theme token (= `0.125rem`), let's the previous value `0.5rem` to keep the look consistent.

```suggestion
    spacing = "0.5rem",
```

Reply:
sry was on autopilot, in our app we are only using theme-token values :)

updated.